package.json

```diff
@@ -4,13 +4,15 @@
   "description": "Make CSS easier and more maintainable by using JavaScript",
   "main": "dist/free-style.js",
   "typings": "dist/free-style.d.ts",
+  "module": "dist.es2015/free-style.js",
```

Comment:
`module` is the field that the JS community seems to be standardizing on, likely to be the name node ends up using. For example: https://github.com/reactjs/redux/blob/e2e5d30741567bbbe64764cbb4d3868c839de5c6/package.json#L6

`jsnext:main` is the field that rollup itself pioneered.